empca.py

```diff
@@ -316,13 +316,19 @@ def empca(data, weights=None, niter=25, nvec=5, smooth=0, randseed=1, silent=Fal
         # print "       iter    chi2/dof     drchi_E     drchi_M   drchi_tot       R2            rchi2"
         print "       iter        R2             rchi2"
     
+    R2_old = 0.
     for k in range(niter):
         model.solve_coeffs()
         model.solve_eigenvectors(smooth=smooth)
+        R2_new = model.R2()
+        R2diff = abs(R2_new-R2_old)
```

Comment:
Could you make this ``math.fabs`` instead of ``abs``; it's better to use the specific ``math`` or ``numpy`` functions than the Python ``abs``.